src/lib/cli/rules/templates/core-rule/rule-test.hbs

```diff
@@ -1,6 +1,6 @@
-/* eslint sort-keys: 0, no-undefined: 0 */
+/* eslint sort-keys: 0 */
```

Comment:
@molant Can we remove this also and change the order of the properties below? Thanks!

Reply:
@molant I did the change before merging.